src/Modules/StatCan.OrchardCore.Hackathon/Manifest.cs

```diff
@@ -20,7 +20,8 @@
         "OrchardCore.Autoroute",
         "OrchardCore.Queries.Sql",
         "OrchardCore.Users.CustomUserSettings",
-        "StatCan.OrchardCore.VueForms.Localized"
+        "StatCan.OrchardCore.VueForms.Localized",
+        "StatCan.OrchardCore.EmailTemplates"
```

Comment:
No need for this here, This feature can be enabled manually and this module does not depend on it. I guess same is true for the VueForms module.